addon/utils/utils.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember'
```

Comment:
Should we find a better name than `utils`? I feel like having a utils class in a utils folder is a bit utility overkill :P